__tests__/components/utils/PatientInfoCard.test.tsx

```diff
@@ -165,4 +166,21 @@ describe('PatientInfoCard', () => {
     const populationSelector = screen.getByPlaceholderText(/select populations/i) as HTMLInputElement;
     expect(populationSelector).toBeInTheDocument();
   });
+
+  it('should render a patient actions menu when the screen is small', () => {
+    render(
+      mantineRecoilWrap(
```

Comment:
Any chance we can use the mock callback props here to make this cleaner? I think we might be able to use them and omit `smallScreen`, similar to what we do for the patient in these unit tests

Reply:
Ah good call!!